src/experiment.py

```diff
@@ -0,0 +1,188 @@
+from optimisers import Problem, Optimiser
+from sensor_group import SensorSuite
+from fields import Field
+import numpy as np
+
+
+
+
+class Experiment():
+    """
+    Manages the optimisers, true field and sensor suite.
```

Comment:
For PEP-257 (https://www.python.org/dev/peps/pep-0257/), please use the imperative mood in docstring summaries, particularly for the first line which should be a single sentence in the imperative mood.
> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

```suggestion
    Manage the optimisers, true field and sensor suite.
```